### DISCUSSION TITLE
WMI No values

### DISCUSSION THREAD

**Discussion by aleohl** [LibreHardwareMonitor/LibreHardwareMonitor#2060] [General]
Running Windows 11 Build 22H2 with LibreHardwareMonitor v0.9.2. Getting no values on WMI query `SELECT Identifier,Name,Value FROM Sensor`. Any ideas?
Found #581 but it seemed to work when @PhyxionNL tested, but it was a while ago.

**Comment by javguit** [comment]
Have you tried * instead of the specific fields?

**Comment by aleohl** [comment]
Yes, sir. I did try with `*`. I just checked with SimpleWMIView and now there is no items at all in WMI Class Sensor or Hardware. Strange. 

Edit: The whole namespace seems to be empty.
Edit 2: Tried with latest nightly and got sensors and hardware items back but still no values.

**Comment by aleohl** [comment]
![2F7CqjTxmf](https://user-images.githubusercontent.com/1404678/230135371-5eba9a60-b2d9-4560-9419-e74a56353d40.jpg)

**Comment by eabase** [comment]
BUMP!

**Comment by Slion** [comment]
To browse WMI I used that tool: https://github.com/vinaypamnani/wmie2
Tried the nightly build and it seems to be working.
